openssh/init.sls

```diff
@@ -5,10 +5,16 @@ openssh:
   pkg.installed:
     - name: {{ openssh.server }}
   {% endif %}
+  {% if openssh.sshd_enable is sameas true %}
```

Comment:
that commit is from long time agon, but I believe this was to handle when someone sets pillar to a string `'true'`, like this:

```yaml
openssh:
   sshd_enable: 'true'
```